src/room.ts

```diff
@@ -130,11 +130,17 @@ export class Room extends EventEmitter implements Sayable {
       const data = await contactGetter(this.id)
       log.silly('Room', `contactGetter(${this.id}) resolved`)
       this.rawObj = data
+      if (!this.rawObj.MemberList) {
+        this.rawObj.MemberList = []
```

Comment:
Do not modify `rawObj`, because this data comes from Browser, which should be treated as _immutable_. 

Do this job on `this.obj` instead.